lib/exzeitable/html/format.ex

```diff
@@ -43,11 +51,15 @@ defmodule Exzeitable.HTML.Format do
   end
 
   # We want socket.assigns, but not socket.assigns.assigns
-  defp smush_assigns_together(socket, assigns) do
-    socket
-    |> Map.fetch!(:assigns)
-    |> Map.delete(:assigns)
-    |> Map.merge(assigns)
+  defp smush_assigns_together(%{socket: socket, params: %{assigns: param_assigns}} = root_assigns) do
+    # there are 3 assigns
+    # root_assigns
+    # root_assigns.params.assigns (from the use Exzeitable)
+    # root_assigns.socket.assigns (always nil)
+
```

Comment:
this was needed because the assigns given to the FIeldFunctions were missing some of the values